src/os/bluestore/BlueFS.cc

```diff
@@ -1322,10 +1322,12 @@ int BlueFS::_truncate(FileWriter *h, uint64_t offset)
   return 0;
 }
 
-void BlueFS::_fsync(FileWriter *h, std::unique_lock<std::mutex>& l)
+int BlueFS::_fsync(FileWriter *h, std::unique_lock<std::mutex>& l)
 {
   dout(10) << __func__ << " " << h << " " << h->file->fnode << dendl;
-  _flush(h, true);
+  int r = _flush(h, true);
+    if (r < 0)
+      return r;
```

Comment:
otherwise this commit is fine.  please prefix teh first line of the git commit message with "os/bluestore/BlueFS: ".